services/notifications/pkg/config/parser/parse.go

```diff
@@ -33,5 +35,22 @@ func ParseConfig(cfg *config.Config) error {
 }
 
 func Validate(cfg *config.Config) error {
+	logger := logging.Configure(cfg.Service.Name, cfg.Log)
+
+	if cfg.Notifications.SMTP.Host != "" {
+		switch cfg.Notifications.SMTP.Encryption {
+		case "tls":
+			logger.Warn().Msg("The smtp_encryption value 'tls' is deprecated. Please use the value 'starttls' instead.")
+		case "ssl":
+			logger.Warn().Msg("The smtp_encryption value 'ssl' is deprecated. Please use the value 'ssltls' instead.")
+		case "starttls", "ssltls", "none":
+			break
+		default:
```

Comment:
Shouldn't we unset the whole smtp configuration if an invalid value is passed here?